src/visitor/AggregateVisitor.h

```diff
@@ -0,0 +1,102 @@
+/* Copyright (c) 2021 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#ifndef VISITOR_AGGREGATEVISITOR_H_
+#define VISITOR_AGGREGATEVISITOR_H_
+
+#include "common/expression/Expression.h"
+#include "visitor/ExprVisitorImpl.h"
+
+namespace nebula {
+namespace graph {
+class AggregateVisitor final : public ExprVisitorImpl {
```

Comment:
Is this visitor only for aggregation, or is there a more general abstraction to it in other rewrite scenarios?